Tests/PennyTests/Tests/LeafRenderTests.swift

```diff
@@ -30,20 +31,20 @@ class LeafRenderTests: XCTestCase {
         )
     )
 
-    override func setUp() async throws {
-        FakeResponseStorage.shared = FakeResponseStorage()
-    }
-
-    override func tearDown() async throws {
-        try await httpClient.shutdown()
+    init() {}
+    
+    deinit {
+        try! httpClient.syncShutdown()
```

Comment:
We should move to a `with` style pattern to manage the lifecycle of the HTTPClient - the deinit pattern is flaky and doesn't play nice with blocking or isolation

Reply:
Just used the shared `HTTPClient` in tests.